client/src/pages/course/admin/students.tsx

```diff
@@ -44,6 +44,7 @@ function Page() {
 
   const [loading, withLoading] = useLoading(false);
   const [hasCourseManagerRole] = useState(isCourseManager(session, courseId));
+  const [hasAdminRole] = useState(isAdmin(session));
```

Comment:
Maybe I misunderstand something, but why do we need `useState` for static thing oO